src/app/tests/suites/DL_LockUnlock.yaml

```diff
@@ -33,6 +33,14 @@ tests:
       command: "UnlockDoor"
       timedInteractionTimeoutMs: 10000
 
+    - label: "Wait for lock action to complete"
```

Comment:
Followup (separate PR): Instead of doing waits then reads, we should just subscribe and wait for reports.

Reply:
I went down this rabbit-hole a while back. Waiting for reports had issues in Yaml (maybe they're fixed now) -- if any other activity came in while waiting for the report, the test runner would get messed up, so there were always race conditions.